tools/python/util/optimize_onnx_model.py

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env python3
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+import argparse
+import os
+import pathlib
+
+from .onnx_model_utils import get_optimization_level, optimize_model
+
+
+def optimize_model_helper():
+    parser = argparse.ArgumentParser(f'{os.path.basename(__file__)}:{optimize_model_helper.__name__}',
+                                     description='''
+                                     Optimize an ONNX model using ONNX Runtime to the specified level.
+                                     See https://onnxruntime.ai/docs/performance/graph-optimizations.html for more
+                                     details of the optimization levels.'''
+                                     )
+
+    parser.add_argument('--opt_level', default='basic',
+                        choices=['disable', 'basic', 'extended', 'all'],
+                        help="Optimization level to use.")
+    parser.add_argument('--log_level', type=int, required=False, default=3,
```

Comment:
>log_level [](http://example.com/codeflow?start=27&length=9)

just a user experience thing - this log_level option is numeric, the log_level option in check_onnx_model_mobile_usability.py is a string. might be nicer if they are consistent or named distinctly

Reply:
Ones for the python logger and one is for the ORT logger. 

I can add some equivalent conversion logic so the user specifies a string in either case.